crates/ra_db/src/input.rs

```diff
@@ -403,6 +403,31 @@ mod tests {
         assert!(graph.add_dep(crate3, CrateName::new("crate1").unwrap(), crate1).is_err());
     }
 
+    #[test]
+    fn it_should_panic_because_of_direct_cycle() {
```

Comment:
```suggestion
    fn it_should_not_panic_because_of_direct_cycle() {
```

Reply:
Hm yeah I was just copying the other test name, not sure if that's the intended interpretation but I guess I'll just give both better names...